model/cicd/spans.yaml

```diff
@@ -0,0 +1,26 @@
+groups:
+  - id: span.cicd.pipeline.run.server
+    type: span
+    span_kind: server
+    brief: 'Semantic Convention for pipeline runs'
+    stability: development
+    attributes:
+      - ref: cicd.pipeline.result
+        requirement_level: required
+  - id: span.cicd.pipeline.task.internal
+    type: span
+    span_kind: internal
+    brief: 'Semantic Convention for tasks in a pipeline run'
+    stability: development
+    attributes:
+      - ref: cicd.pipeline.task.name
```

Comment:
please add error.type

Reply:
Done in aa5e4771d592943d8e3caa8750b898e408ff66bd.